docs/welcome/releases.md

```diff
@@ -1,32 +1,10 @@
 
 # kOps Releases & Versioning
 
-kOps intends to be backward compatible.  It is always recommended using the
-latest version of kOps with whatever version of Kubernetes you are using.  We suggest
-kOps users run one of the [3 minor versions](https://kubernetes.io/releases/version-skew-policy/#supported-versions) Kubernetes is supporting however we
-do our best to support previous releases for some period.
+kOps supports the latest minor version and latest-1. E.g if kOps' latest version is 1.25, also 1.24 is supported and will receive bugfixes and minor feature additions. kOps intends to be backward compatible.  It is always recommended using the
+latest version of kOps regardless of what Kubernetes version you are using.  
```

Comment:
to avoid suggesting kOps 1.25 on k8s 1.26 before kOps 1.26 is released:
```suggestion
latest version of kOps that supports the Kubernetes version you are using.  
```